src/test_c_kzg_4844.c

```diff
@@ -1089,17 +1089,27 @@ static void test_compute_kzg_proof__succeeds_expected_proof(void) {
     /* Compare the computed proof to the expected proof */
     diff = memcmp(proof.bytes, expected_proof.bytes, sizeof(Bytes48));
     ASSERT_EQUALS(diff, 0);
+
+    bytes32_from_hex(
```

Comment:
Perhaps another approach here would be to evaluate the blob at `input_value` using `evaluate_polynomial_in_evaluation_form` and make sure the output matches what compute_kzg_proof returned.